tensorrt_llm/_torch/auto_deploy/custom_ops/linear/swiglu.py

```
@@ -93,6 +94,57 @@ def _(
     return input.new_empty(output_shape, dtype=input.dtype)
 
 
+# ── Triton backend ───────────────────────────────────────────────────────────
+
+from .triton_swiglu import swiglu_mlp as _triton_swiglu_mlp  # noqa: E402
+
+
+@torch.library.custom_op("auto_deploy::triton_swiglu_mlp", mutates_args=())
+def triton_swiglu_mlp(
+    input: torch.Tensor,
```

Ruff (0.15.6)

[error] Function argument `input` is shadowing a Python builtin

(A002)